test-data/unit/fixtures/bool.pyi

```diff
@@ -10,9 +10,11 @@ class object:
 class type: pass
 class tuple(Generic[T]): pass
 class function: pass
-class int: pass
+class int:
```

Comment:
Similarly, it seems like we only should need fixture changes in fixtures/list.pyi